src/StatCache.cc

```diff
@@ -33,7 +33,7 @@
 
 namespace {
 
-const float stale = 0.190;  // Maximum lifetime of a cache entry chosen to be shorter than the automatic reload poll time
+const float stale = 190;  // Maximum lifetime of a cache entry chosen to be shorter than the automatic reload poll time
```

Comment:
Perhaps mention "milliseconds" here, so that when we eventually move to C++14 we can use the typesafe time operator and just write `constexpr auto stale = 190ms;`,